src/l1-resources/cloud-backup-restore-jobs/index.ts

```diff
@@ -16,14 +16,14 @@ export interface CfnCloudBackUpRestoreJobsProps {
   readonly projectId: string;
 
   /**
-   * The name of the Atlas cluster whose snapshot you want to restore or you want to retrieve restore jobs.
+   * Type of instance specified on the Instance Name serverless or cluster
    *
-   * @schema CfnCloudBackUpRestoreJobsProps#ClusterName
+   * @schema CfnCloudBackUpRestoreJobsProps#InstanceType
    */
-  readonly clusterName: string;
+  readonly instanceType?: CfnCloudBackUpRestoreJobsPropsInstanceType;
```

Comment:
isn't instanceType mandatory in CFN?

Reply:
is defined within the CreareRequiredFields, but not in the schema, ill create a task to fix that on cfn